frontend/src/router/pages/SubmitDates.jsx

```diff
@@ -0,0 +1,230 @@
+import { useParams } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import groupCamping from "../../assets/icons/group-camping.svg";
+
+import "./SubmitDates.scss";
+
+function SubmitDates() {
+  // @TODO: fetch park name + details
+  const { parkId, seasonId } = useParams();
+
+  function SubArea() {
+    return (
+      <section className="sub-area">
+        <h3>Alouette groupsite</h3>
+
+        <div className="row">
+          <div className="col-md-6">
+            <p>
+              Operating dates{" "}
+              <FontAwesomeIcon
+                className="append-content"
+                icon="fa-solid fa-circle-info"
+              />
```

Comment:
This icon in the mock seems to be a regular one (white background). Also, we need to add some text to these tooltips at some point.

Reply:
Yeah 😮‍💨 a mix of two different icon sets.. I'm dreading setting up a "Pro icon kit" for FontAwesome here but I think that's what we'll have to do. When we build out the actual React components, we can use the react-bootstrap tooltip component ✅ 

I'll leave the icon with a placeholder until we sort out the tooltip content 👍